src/SteamAuth.php

```diff
@@ -49,10 +49,13 @@ class SteamAuth implements SteamAuthInterface
      *
      * @param Request $request
      */
-    public function __construct(Request $request)
+    public function __construct(Request $request, $redirect_url = null)
```

Comment:
This class is bind as singleton in https://github.com/invisnik/laravel-steam-auth/blob/master/src/SteamServiceProvider.php#L32
If you want to customize the redirect-url you should adjust the `redirect` method.

```php
public function redirect($redirectUrl = null)
{
    return redirect(is_null(redirectUrl) ? $this->getAuthUrl() : $this->buildUrl(url($redirectUrl)));
}
```